README.md

```diff
@@ -1,33 +1,41 @@
-[![License](https://img.shields.io/github/license/ginkgo-project/ginkgo.svg)](./LICENSE)
-[![c++ standard](https://img.shields.io/badge/c%2B%2B-14-blue.svg)](https://en.wikipedia.org/wiki/C%2B%2B#Standardization)
-[![DOI](https://joss.theoj.org/papers/10.21105/joss.02260/status.svg)](https://doi.org/10.21105/joss.02260)
-
-Ginkgo is a high-performance linear algebra library for manycore systems, with a
-focus on the solution of sparse linear systems. It is implemented using modern C++
-(you will need an at least C++14 compliant compiler to build it), with GPU kernels
-implemented in CUDA, HIP, and DPC++(SYCL).
+  <p align="center"><img src="/assets/logo.png" alt="Ginkgo" width="60%" height="60%"></p>
+
+<div align="center">
+
+License|C++ Standard|Documentation|JOSS paper
+:--:|:--:|:--:|:--:
+[![License](https://img.shields.io/github/license/ginkgo-project/ginkgo.svg)](./LICENSE)|[![c++ standard](https://img.shields.io/badge/c%2B%2B-14-blue.svg)](https://en.wikipedia.org/wiki/C%2B%2B#Standardization)|[![Documentation](https://img.shields.io/badge/Documentation-latest-blue.svg)](https://ginkgo-project.github.io/ginkgo-generated-documentation/doc/develop/)|[![DOI](https://joss.theoj.org/papers/10.21105/joss.02260/status.svg)](https://doi.org/10.21105/joss.02260)
+
+
+CI builds|Coverage| Maintainability|Reliability|Dashboard
+:--:|:--:|:--:|:--:|:--:
+[![Build status](https://gitlab.com/ginkgo-project/ginkgo-public-ci/badges/develop/pipeline.svg)](https://github.com/ginkgo-project/ginkgo/commits/develop)[![OSX-build](https://github.com/ginkgo-project/ginkgo/actions/workflows/osx.yml/badge.svg)](https://github.com/ginkgo-project/ginkgo/actions/workflows/osx.yml)[![Windows-build](https://github.com/ginkgo-project/ginkgo/actions/workflows/windows-msvc-ref.yml/badge.svg)](https://github.com/ginkgo-project/ginkgo/actions/workflows/windows-msvc-ref.yml)|[![codecov](https://codecov.io/gh/ginkgo-project/ginkgo/branch/develop/graph/badge.svg)](https://codecov.io/gh/ginkgo-project/ginkgo)|[![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=ginkgo-project_ginkgo&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=ginkgo-project_ginkgo)|[![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=ginkgo-project_ginkgo&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=ginkgo-project_ginkgo)|[![CDash dashboard](https://img.shields.io/badge/CDash-Access-blue.svg)](https://my.cdash.org/index.php?project=Ginkgo+Project)
 
+</div>
 
-Performance
+Ginkgo is a high-performance numerical linear algebra library for many-core systems, with a
+focus on solution of sparse linear systems. It is implemented using modern C++
+(you will need an at least C++14 compliant compiler to build it), with GPU kernels
+implemented for NVIDIA, AMD and Intel GPUs.
+
+Table of Contents
```

Comment:
Some headings use the `#` syntax and some use the `-------` syntax. Maybe just use the `#` one everywhere.